### PR TITLE
Update INSTALL.md

### PR DESCRIPTION
**Pull request by sidey79**
Added hint for CMAKE_INSTALL_PREFIX  and CMAKE_PREFIX_PATH

### CODE REVIEW

**Comment by end2endzone** [comment]
I have take a look at your modifications. There are some small typos that I will fix right after applying your modifications. I also see what you mean before... I was unfamiliar with `CMAKE_PREFIX_PATH` and find it really useful. I learned something today! hehehe thanks.

**Comment by sidey79** [comment]
thanks for that.

May you know it, but i found a very interesting cmake command:
https://cmake.org/cmake/help/git-stage/module/ExternalProject.html

i think with this it is possbile to download from a git repository and install it somewhere. So no extra need of manual download via git.

File: INSTALL.md [hunk 1]
```diff
@@ -64,6 +61,11 @@ mkdir build
 cd build
 cmake ..
 ```
+If you do want to specify a specific location, where rapid assist should be installed, you can provide an extra option to cmake at this step `-DCMAKE_INSTALL_PREFIX=c:\projects\install\rapidassist`
+If you do not spefiy this, cmake will install that library into its default location, which differs from system to system. On Windows it is `c:\progam files (x86)\rapidassis` and for this admin rights are needed.
+
+If you choos to install your libraries into a separate folder, cmake doen't know where to search for rapidassist, when you try to build another project, win32arduino for example.
```
Comment (sidey79) [inline INSTALL.md:67]: typo: cmake doesn't know

**Comment by end2endzone** [comment]
I notice you changed the indentation below the 3rd bullet (at level 2 ) by changing from `1)` to `1a)`. This is not valid markdown and break second level. You can take a look [here](https://github.com/end2endzone/RapidAssist/blob/636d1d447b18ddc3404df6f21c283f2071cb0a23/INSTALL.md). I took the liberty to change it back to `1)`. By the way, you can validate your markdown changes using this [online markdown editor](https://jbt.github.io/markdown-editor/) which supports github's own flavor.

**Comment by end2endzone** [comment]
Changes applied to 7d680d19e122118449e34cf6c2fe45911372bdbd

**Comment by end2endzone** [comment]
Yes I am already aware of the `ExternalProject` command. It is true that you can use this command to download a project's dependencies from a git repository. It is also a nice "portable" way of doing it. I used to do the same (almost) with git submodules but it was more complicated. The `ExternalProject` command is nicer.

However, I prefer to have the instructions say  "build all the dependencies according to their instructions" rater than having to copy and paste how to build (for instance) rapidassist in all projects where rapidassist is used as a dependency.

**Pull request merged**